cmd/data.go

```diff
@@ -165,21 +165,31 @@ func (cmd *DataCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface
 		}
 	}
 
-	dataCoversionStartTime := time.Now()
-	bw, err := conversion.DataConv(ctx, sourceProfile, targetProfile, &ioHelper, client, conv, true, cmd.writeLimit)
+	streamingCfg, err := startDatastream(ctx, sourceProfile, targetProfile)
```

Comment:
The code looks the same here and and schema_and_data, we should probably call this whole block as migrateData or so and call a common method in both places.